### PR TITLE
Update boards content for 7.10

### PR DESCRIPTION
**Pull request by justinegeffen**
Content related to: https://github.com/mattermost/mattermost-server/pull/22195

### CODE REVIEW

File: source/boards/getting-started.rst [hunk 1]
```diff
@@ -10,14 +10,14 @@ Mattermost Boards (formerly known as Focalboard) is accessible from the Product
    
   .. tab:: Mattermost v6.0 onwards
    
-      If you're using Mattermost 6.0, select the Product menu in the top left corner of Mattermost to activate Boards. The Boards dashboard is where you can get
+      If you're using Mattermost 6.0, select the Product menu in the top left corner of Mattermost to open Boards. The Boards dashboard is where you can get
       started, where you'll find all the boards you have access to, and also where you can navigate between workspaces. Once you've activated Boards, you'll see a tab at the top of Mattermost called **Boards**. Select that for quick access to your dashboard.
   
   .. tab:: Mattermost v5.39 and earlier
 
       For self-hosted deployments using versions 5.38 and earlier, Boards is available in the Marketplace.
```
Comment (justinegeffen) [inline source/boards/getting-started.rst:18]: @wuwinson, is this still applicable if the product is no longer a plugin?

**Comment by justinegeffen** [comment]
@wuwinson, in the sharing content we provide this process:

Sharing boards publicly is disabled by default. This means that the **Publish** tab is not available from the **Share** dialog. To enable public board sharing:

1. Go to **Product menu > System Console > Plugins > Plugin Management > Mattermost Boards > Settings**.
2. Set **Enable Publicly-Shared Boards** to **true**.
3. Select **Save**.

Does this change at all? Should we call out different behavior across versions?

**Review by wuwinson — COMMENTED** [review]
@justinegeffen I also removed the section on "dashboards" which was deprecated as of v7.2.

**Comment by wuwinson** [comment]
> 



> @wuwinson, in the sharing content we provide this process:
> 
> Sharing boards publicly is disabled by default. This means that the **Publish** tab is not available from the **Share** dialog. To enable public board sharing:
> 
> 1. Go to **Product menu > System Console > Plugins > Plugin Management > Mattermost Boards > Settings**.
> 2. Set **Enable Publicly-Shared Boards** to **true**.
> 3. Select **Save**.
> 
> Does this change at all? Should we call out different behavior across versions?

@justinegeffen Yes, good catch! Step 1 would change to:
Go to **Product menu > System Console > Products > Boards**.

Screen Shot 2023-04-11 at 1 05 40 PM

File: source/boards/share-and-collaborate.rst [hunk 2]
```diff
@@ -117,7 +117,7 @@ Share a board publicly
 
 Sharing boards publicly is disabled by default. This means that the **Publish** tab is not available from the **Share** dialog. To enable public board sharing:
 
-1. Go to **Product menu > System Console > Plugins > Plugin Management > Mattermost Boards > Settings**.
+1. Go to **Product menu > System Console > Products > Boards > Settings**.
```
Comment (wuwinson) [inline source/boards/share-and-collaborate.rst:120]:
```suggestion
1. Go to **Product menu > System Console > Products > Boards**.
```

**Review by wuwinson — APPROVED** [review]
Made some minor tweaks. Everything else looks good!

**Pull request merged**